problem_builder/step_review.py

```diff
@@ -262,6 +265,7 @@ def student_view(self, context=None):
             # Note: The following text should never be seen (in theory) so does not need to be translated.
             fragment.add_content(u"Your score and review messages will appear here.")
         else:
+            has_per_q_feedback_block = False
```

Comment:
@bradenmacdonald Nit: I'd probably go for `has_per_question_feedback_block` here. It's clear from context what `q` stands for, but since there aren't any abbreviations in the surrounding code, this sticks out, kind of :)